.eslintrc

```diff
@@ -0,0 +1,56 @@
+{
+  "ecmaFeatures": {
+    "jsx": true
+  },
+
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "node": true
+  },
+
+  "plugins": [
+    "react"
+  ],
+
+  "globals": {
+    "jest": true,
+  },
+
+  "rules": {
+    "quotes": [2, "double"],
+    "indent": [2, 2],
+    "camelcase": false,
+    "strict": [2, "never"], // <-- fix
+    "no-mixed-spaces-and-tabs": [2, "always"],
+    "no-trailing-spaces": [2, "always"],
+    "no-underscore-dangle": false,
+    "no-multiple-empty-lines": [2, {"max": 1}],
+    "space-after-keywords": [2, "always"],
+    "space-before-blocks": [2 "always"],
+    // "space-before-function-paren": [2, {
+    //   "anonymous": "always", "named": "never"
+    // }], TODO: DCOS-914
```

Comment:
I suggest you either fix these outstanding requirements, or create Marathon UI tickets for them